engines/python/setup/djl_python/rolling_batch/rolling_batch_vllm_utils.py

```diff
@@ -269,19 +269,21 @@ def get_engine_args_from_config(config: VllmRbProperties) -> EngineArgs:
         )
 
 
-def get_multi_modal_data(request: Request) -> dict:
+def get_multi_modal_data(request: Request, config: VllmRbProperties) -> dict:
```

Comment:
should we pass in the config.limi_mm_per_prompt instead of the whole config?

Reply:
Changed.